ItemLevelDownloader/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "item-level-downloader",
-  "version": "1.0.10",
+  "version": "1.0.11",
```

Comment:
update package version before checkin.

Reply:
i have updated it here to 1.0.11, is there some other version also that needs to be changed ?